docs/content/topics/scripting/guide.md

```diff
@@ -9,1401 +9,867 @@ aliases:
+using the [brig] CLI. See the [Quickstart] if you have not already done so.
 
-Based on the above, we have:
+Then, each example project can be created like so:
 
-- A project named `brigadecore/empty-testbed`
-- A GitHub repo at `https://github.com/brigadecore/empty-testbed.git`
-- And a single secret: `dbPassword: supersecret`
+```console
+$ brig project create -f examples/<project>/project.yaml
+```
 
-We'll use this project throughout the tutorial as we create some `brigade.js` files and test
-them out.
+Let's begin!
 
-We will be using the `brig` binary to execute our brigade scripts.  Usage
-and installation instructions for `brig` are [here](/brig/).
+[quickstart]: /topics/intro/quickstart
+[project]: /topics/project-developers/projects
+[`package.json`]: /topics/scripting/dependencies
+[secrets]: /topics/project-developers/secrets
+[project definition]: /topics/project-developers/projects#project-definition-files
+[example projects]: https://github.com/brigadecore/brigade/tree/v2/examples
+[brig]: /topics/project-developers/brig
 
 ## A Basic Brigade Script
 
-Here is a basic `brigade.js` file that just sends a single
-message to the log:
+Here is a basic `brigade.js` script that just sends a single message to the
+log:
 
 ```javascript
-console.log("hello world")
+console.log("Hello, World!");
 ```
-[brigade-01.js](../../examples/brigade-01.js)
+[01-hello-world](https://github.com/brigadecore/brigade/tree/v2/examples/01-hello-world)
 
-If we run this script, we would see output that looked something like this:
+First let's create the example project:
 
 ```console
-Started brigade-worker-01brwzs64rve2jvky87hxy1wsp-master
-yarn start v0.27.5
-$ node prestart.js
-prestart: src/brigade.js written
-$ node --no-deprecation ./dist/src/index.js
-hello world
-Done in 1.44s.
+$ brig project create --file examples/01-hello-world/project.yaml
+
+Created project "hello-world".
 ```
 
-> Tip: You can use the `brig`  command to send brigade.js
-> files to Brigade.
->
-> In this tutorial we run scripts with `brig run --file brigade.js $PROJECT`, where `$PROJECT` is a
-> project ID like `brigade-830c16d4aaf6f5490937ad719afd8490a5bcbef064d397411043ac`.
+Next, let's trigger execution of the project script by creating an event:
 
-In essence, all we have done is started a brigade session, logged "hello world", and exited.
+```console
+$ brig event create --project hello-world --follow
 
-This example executes the log in the _global scope_. While it's fine to work in the
-global scope, most of the time what we want to do with Brigade is write _event handlers_.
+Created event "261229dc-1140-4f6a-bf91-bd2a69f31721".
+
+Waiting for event's worker to be RUNNING...
+2021-09-20T22:15:12.047Z INFO: brigade-worker version: a398ba8-dirty
+Hello, World!
+```
+
+This example executes the log in the _global scope_. While it's fine to work
+in the global scope, most of the time what we want to do with Brigade is write
+_event handlers_.
```

Comment:
"Global scope" could be confusing. What would you think about saying that this script _unconditionally_ logs "Hello, World!" in response to _any_ event to which the project is subscribed and that it's more common to in corporate event handlers to handle specific events in different ways?